### PR TITLE
Localization system

### PR DESCRIPTION
**Pull request by Entwistle-905**
Set all the languages we need (En, Pt, Ko, Gu, Hi)
Added fonts we need
Set the font by languages 
Create a base game mode for controlling both game modes we have
little bit of Korean translation

### CODE REVIEW

File: Source/Redemption/Private/BaseLocalizationWidget.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+void UBaseLocalizationWidget::NativeConstruct()
+{
+	Super::NativeConstruct();
+}
+
+UFont* UBaseLocalizationWidget::SetFontPerLanguage() const
+{
+	FString CurrentLanguage = FInternationalization::Get().GetCurrentCulture()->GetName();
```
Comment (Fedic) [inline Source/Redemption/Private/BaseLocalizationWidget.cpp:21]: This is the sole way of changing the language correct?

File: Source/Redemption/Private/FontManager.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	{
+		return HindiFont;
+	}
+	else if (Lang == "gu")
+	{
+		return GujaratiFont;
+	}
+	else
+	{
+		UE_LOG(LogTemp, Warning, TEXT("There is no language you want."));
```
Comment (Fedic) [inline Source/Redemption/Private/FontManager.cpp:52]: Grammar wise, I think something like, "That language is unavailable".

File: Source/Redemption/Private/BaseLocalizationWidget.cpp (hunk 1 above)
Comment (mac2648) [reply Source/Redemption/Private/BaseLocalizationWidget.cpp:21]: this is to get the current language, not change it.

**Review by mac2648 — APPROVED** [review]
creating new classes to have the shared functionality, and then changing the parent class of classes is really good design

File: Source/Redemption/Private/FontManager.cpp (hunk 2 above)
Comment (Entwistle-905) [reply Source/Redemption/Private/FontManager.cpp:52]: Thank you haha

**Pull request merged**